src/main/java/de/komoot/photon/App.java

```diff
@@ -64,8 +64,17 @@ public static void main(String[] rawArgs) throws Exception {
                 return;
             }
 
-            // no special action specified -> normal mode: start search API
-            startApi(args, esClient);
+            if (args.isNominatimUpdate()) {
+                shutdownES = true;
+                log.info("Ensuring that the cluster is ready, this might take some time.");
+                // inspired by https://stackoverflow.com/a/50316299
+                esClient.admin().cluster().prepareHealth().setWaitForGreenStatus().execute().actionGet();
+                final NominatimUpdater nominatimUpdater = setupNominatimUpdater(args, esClient);
+                nominatimUpdater.update();
+            } else {
```

Comment:
Please keep with the style of the other ifs and do the `return;` instead of the `else`.